manage_breast_screening/mammograms/jinja2/mammograms/record_medical_information.jinja

```diff
@@ -3,65 +3,101 @@
 {% from "nhsuk/components/card/macro.jinja" import card %}
 {% from "nhsuk/components/inset-text/macro.jinja" import insetText %}
 {% from "nhsuk/components/summary-list/macro.jinja" import summaryList %}
+{% from 'nhsuk/components/tag/macro.jinja' import tag %}
 {% from "components/participant-details/summary_list_rows.jinja" import last_mammogram_html %}
 
 {% block step_content %}
 
-{% call card({"heading": "Mammogram history"}) %}
-  <p>The last confirmed mammogram and any added manually since then</p>
+<div id="{{ presenter.get_anchor(sections.MAMMOGRAM_HISTORY) }}" class="app-reviewable-section">
```

Comment:
I don't know if this is reflected in a ticket anywhere, but we will want to update the success redirect URLs so that they include this fragment as well. That way you don't lose your place when you add things via the forms linked from this page.